pkg/query-service/model/response.go

```diff
@@ -269,6 +269,32 @@ type SearchSpanResponseItem struct {
 	SpanKind         string            `json:"spanKind"`
 }
 
+type SearchSpanResponseItemV2 struct {
+	TimeUnixNano      time.Time          `json:"timestamp" ch:"timestamp"`
+	DurationNano      uint64             `json:"durationNano" ch:"duration_nano"`
+	SpanID            string             `json:"spanId" ch:"span_id"`
+	TraceID           string             `json:"traceId" ch:"trace_id"`
+	HasError          bool               `json:"hasError" ch:"has_error"`
+	Kind              int8               `json:"kind" ch:"kind"`
+	ServiceName       string             `json:"serviceName" ch:"resource_string_service$$name"`
+	Name              string             `json:"name" ch:"name"`
+	References        string             `json:"references,omitempty" ch:"references"`
```

Comment:
We store references a  string only right now.

Reply:
Why does how you store it in DB dictate how it's sent in response? Just run the API once and see the output yourself.

Reply:
This is how references were returned previously
```
[
    "{TraceId=000000000000000065edb2f6f0d99672, SpanId=, RefType=CHILD_OF}"
],
```

And this is how we are returning it now
```
[
    "{TraceId=000000000000000020f6fff780d13b0c, SpanId=4e1a07fd6fd54b69, RefType=CHILD_OF}"
],
```

I am not sure what you are trying to say. I wanted to avoid frontend changes so I am going with keeping the API response same.

Reply:
Sorry for the confusion, it's not the actual response item, I have removed the json tags and moved it to a separete file. Thanks for pointing out.